### PR TITLE
Port 4485 shrink ocean images

### PR DESCRIPTION
**Pull request by yairsimantov20**
# Description

What - Each ocean integration size is now 1.2 GB
Why - A Lot of unneeded files in the dockerfile
How - Removed the venv and unneeded kafka files and cache files

## Type of change

- [X] Non-breaking change (fix of existing functionality that will not change current behavior)

### CODE REVIEW

File: changelog/PORT-4485.improvement.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Shrunk the docker file image result size
```
Comment (MPTG94) [inline changelog/PORT-4485.improvement.md:1]:
```suggestion
Optimized dockerfile to produce smaller images
```

File: integrations/gitlab/CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,14 @@ this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 <!-- towncrier release notes start -->
 
+0.1.8 (2023-08-11)
+==================
+
+### Improvements
+
+- Using a more slim version of the docker file (PORT-4485)
```
Comment (MPTG94) [inline integrations/gitlab/CHANGELOG.md:15]:
```suggestion
- Optimized dockerfile to produce smaller images (PORT-4485)
```

File: integrations/jira/CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 <!-- towncrier release notes start -->
 
+0.1.2 (2023-08-11)
+
+### Improvements
+
+- Using a more slim version of the docker file (PORT-4485)
```
Comment (MPTG94) [inline integrations/jira/CHANGELOG.md:14]:
```suggestion
- Optimized dockerfile to produce smaller images (PORT-4485)
```

File: integrations/newrelic/CHANGELOG.md [hunk 4]
```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 <!-- towncrier release notes start -->
 
+# Port_Ocean 0.1.2 (2023-08-11)
+
+### Improvements
+
+- Using a more slim version of the docker file (PORT-4485)
```
Comment (MPTG94) [inline integrations/newrelic/CHANGELOG.md:14]:
```suggestion
- Optimized dockerfile to produce smaller images (PORT-4485)
```

File: integrations/newrelic/pyproject.toml [hunk 5]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "newrelic"
-version = "0.1.0"
+version = "0.1.2"
```
Comment (MPTG94) [inline integrations/newrelic/pyproject.toml:3]: Why 2 patch versions bump?

File: integrations/pagerduty/CHANGELOG.md [hunk 6]
```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 <!-- towncrier release notes start -->
 
+# Port_Ocean 0.1.4 (2023-08-11)
+
+### Improvements
+
+- Using a more slim version of the docker file (PORT-4485)
```
Comment (MPTG94) [inline integrations/pagerduty/CHANGELOG.md:14]:
```suggestion
- Optimized dockerfile to produce smaller images (PORT-4485)
```

**Review by MPTG94 — APPROVED** [review]
LGTM

File: integrations/newrelic/pyproject.toml (hunk 5 above)
Comment (yairsimantov20) [reply integrations/newrelic/pyproject.toml:3]: Tom forgot to update this file along with spec.yml

**Pull request merged**